docs/core/dependency-loading/loading-managed.md

```diff
@@ -56,8 +58,7 @@ The following algorithm describes how the runtime loads a managed assembly.
 
     - Loading from the specified path or raw assembly object.
 
+      - If an assembly is newly loaded, a reference is added to `active` <xref:System.Runtime.Loader.AssemblyLoadContext> instance's `cache-by-name`.
+
 4. In either case, if an assembly is newly loaded, then:
    - The <xref:System.AppDomain.AssemblyLoad?displayProperty=nameWithType> event is raised.
-   - A reference is added to the assembly's <xref:System.Runtime.Loader.AssemblyLoadContext> instance's `cache-by-name`.
```

Comment:
Right. I think this line and the next were definitely confusing. One could easily assume that assembly name->ref is cached by every ALC involved.